### DISCUSSION TITLE
HERE Place Search Plugin

### DISCUSSION THREAD

**Discussion by jstride** [MM2-0/Kvaesitso#1147] [Q&A] [answered]
I have added the API key but I don't seem to get any results for nearby places when searching. I cannot see any errors in the logs. Is there a trick to getting this working? I have increased the search radius too.

**Comment by MM2-0** [comment, marked as answer]
Make sure that
- the plugin is enabled
- the launcher has been granted location permission
- online search results are enabled

**Comment by jstride** [comment]
Which option is online search results, I don't seem to have it. My version is installed from github releases, would that make a difference?

I've attached a few screenshots. Showing Here app with results but nothing in the launcher.

-------- Original Message --------
